rotkehlchen/data/importer.py

```diff
@@ -212,7 +214,11 @@ def import_cointracking_csv(
         with open(filepath, 'r', encoding='utf-8-sig') as csvfile:
             data = csv.reader(csvfile, delimiter=',', quotechar='"')
             header = remap_header(next(data))
+            imported_rows = 0
             for row in data:
+                imported_rows += 1
```

Comment:
Why do this? You can simply enumerate.

```python
for idx, row in enumerate(data):
```

and use the `idx` for context switching